tests/auth_change_password_test.py

```diff
@@ -0,0 +1,224 @@
+"""Project: New Arrivals Chi.
+
+File name: auth_change_password_test.py
+
+Associated Files:
+    Templates: change_password.html, login.html, profile.html.
+
+This test suite verifies the functionality of the change password feature,
+including testing various scenarios such as incorrect old password,
+new password validation, password confirmation mismatch, and successful
+password change.
+
+Methods:
+    * test_access_change_password_page
+    * test_change_password_wrong_old_password
+    * test_change_password_wrong_new_password_same_as_old
+    * test_change_password_new_passwords_do_not_match
+    * test_change_password_new_password_invalid
+    * test_change_password_success
+
+Last updated:
+@Author: Kathryn Link-Oberstar @klinkoberstar
+@Date: 05/08/2024
+
+Creation:
+@Author: Kathryn Link-Oberstar @klinkoberstar
+@Date: 05/07/2024
+"""
+
+
+def test_access_change_password_page(
+    client, capture_templates, test_user, login_client
+):
+    """Test Access Change Password Page.
+
+    Verifies that the change password page is accessible.
+    Ensures that the correct template is rendered when accessing this page.
+
+    Args:
+        client: The test client used for making requests.
+        capture_templates: Context manager to capture templates rendered.
+        test_user: User instance for which the test is run.
+        login_client: Session object for the logged-in client.
+    """
+    response = client.get("/change_password", follow_redirects=True)
```

Comment:
should this be `login_client`?

Reply:
The way I have it set up, the login_client just logs the test user in at the start of the test, it's not a separate logged-in instance of the app so it would still be client.get

realizing the name is confusing - in my head it was like  login_client = "log into the client" which is not at all how it reads here. i can change the naming to make it less confusing!

Reply:
I see that makes sense! You can keep the naming, just add explanation to `readme` for the new fixtures.

Reply:
ended up changing the name bc i think it is confusing - updated the readme as well!